ajax/libs/native-promise-only/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "native-promise-only",
+  "filename": "npo.js",
+  "author": {
+    "email": "getify@gmail.com",
+    "name": "Kyle Simpson"
+  },
```

Comment:
Yes, since we don't have policy on this field, there is no need to manutally modify that to be difference with upstream.